Tests/ResponseSerializationTests.swift

```diff
@@ -22,15 +22,15 @@
 //  THE SOFTWARE.
 //
 
+@testable
 import Alamofire
```

Comment:
Removed the `@testable`, as I moved the boolean extensions into the test target.